optimum/exporters/neuron/convert.py

```diff
@@ -426,20 +426,42 @@ def export_neuronx(
         compiler_args = ["--auto-cast", "none"]
 
     # diffusers specific
-    if hasattr(config._config, "_class_name") and "unet" in config._config._class_name.lower():
-        compiler_args.extend(["--model-type", "unet-inference"])
+    prepare_stable_diffusion_compilation(config, compiler_args)
 
     neuron_model = neuronx.trace(checked_model, dummy_inputs_tuple, compiler_args=compiler_args)
 
     if config.dynamic_batch_size is True:
         neuron_model = neuronx.dynamic_batch(neuron_model)
 
+    # diffusers specific
+    improve_stable_diffusion_loading(config, neuron_model)
+
     torch.jit.save(neuron_model, output)
     del neuron_model
 
     return config.inputs, config.outputs
 
 
+def prepare_stable_diffusion_compilation(config, compiler_args):
+    if hasattr(config._config, "_name_or_path"):
+        sd_components = ["text_encoder", "unet", "vae", "vae_encoder", "vae_decoder"]
+        if any(component in config._config._name_or_path.lower() for component in sd_components):
+            compiler_args.extend(["--enable-fast-loading-neuron-binaries"])
+        # unet
+        if "unet" in config._config._name_or_path.lower():
+            compiler_args.extend(["--model-type=unet-inference"])
```

Comment:
nit: I'd rather have a function that creates a new `compile_args` with the additional flags than a function modifying in place.